entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -eo pipefail
+
+LOCUST_BIN='/usr/bin/locust'
```

Comment:
There's a functional difference between single quotes and double quotes - no variable interpolation is done within single quotes. I prefer to use these accordingly (and believe it's a good practice).

Reply:
Yes, in my experience the functional difference is this:

* (with double quotes) "I need to add a variable to this string. Done."

* (with single quotes) "I need to add a variable to this string. What, it didn't work? Oh, because I need to change it to double quotes."

For this reason, I prefer double quotes everywhere (except when they cannot be used). Most of our scripts (and most shell scripts on the internet) follow this pattern.

Reply:
Hm, this is probably for longer discussion and highly opinionated. I agree with the argument for being consistent, but looking at our code you can find lot of things like:
- `num_failures=$(echo $SESSION_STATS | jq -r '.stats[] | select(.name == "Total").num_failures')`
- `-X POST -H 'Content-Type: application/json' -d '{"action": "finish_cluster"}')`
- `--user=$(gcloud info --format='value(config.account)')`

Anyway my thinking on how to improve this in long-term -> a) create a "style guide" to agree and clarify how this should look like and b) add some linter (popular one is https://github.com/koalaman/shellcheck) to our tests. What do you think?

Reply:
I agree this leads to a larger discussion. I agree that a style guide and a linter are the "real" solutions for this problem. However I think we generally use double quotes throughout the codebase pretty consistently.

The quoted examples look correct to me, because they are using single quotes to enable (unescaped) embedding of double quotes. (I agree that the last one should be double quotes unless there is some other consideration.)